### PR TITLE
Drop support for iOS 12 and 13

### PR DESCRIPTION
**Pull request by NickEntin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ Want to log with Aardvark but don’t want to use Aardvark’s bug reporting too
 
 ## Requirements
 
-* Xcode 11.0 or later
```
Comment (dfed) [inline README.md:99]: `ci.yml` is still using Xcode_11.7. We probably wanna update that 🙂

Comment (NickEntin) [reply README.md:99]: 🤔 Pretty sure it was using Xcode 13. In any case, should be Xcode 14 now. 😄

Comment (dfed) [reply README.md:99]: 🤦‍♂️ apparently I was quite behind. My bad!

**Review by dfed — APPROVED** [review]

**Comment by NickEntin** [comment]
```
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "Headers/AardvarkMailUI.h"
        ^
/Users/runner/Library/Developer/Xcode/DerivedData/AardvarkSample-awjiniafbciuuqbtnjpwgcqgttcb/Build/Products/Debug-iphonesimulator/AardvarkMailUI.framework/Headers/AardvarkMailUI.h:28:9: note: in file included from /Users/runner/Library/Developer/Xcode/DerivedData/AardvarkSample-awjiniafbciuuqbtnjpwgcqgttcb/Build/Products/Debug-iphonesimulator/AardvarkMailUI.framework/Headers/AardvarkMailUI.h:28:
#import <AardvarkMailUI/ARKEmailBugReporter.h>
        ^
/Users/runner/Library/Developer/Xcode/DerivedData/AardvarkSample-awjiniafbciuuqbtnjpwgcqgttcb/Build/Products/Debug-iphonesimulator/AardvarkMailUI.framework/Headers/ARKEmailBugReporter.h:67:44: error: no type or protocol named 'ARKBugReporter'
@interface ARKEmailBugReporter : NSObject <ARKBugReporter>
                                           ^
<unknown>:0: error: could not build Objective-C module 'AardvarkMailUI'
```

But... why? I'm guessing this must be some subtle change in Swift/ObjC interop.

**Comment by NickEntin** [comment]
```
    xcodebuild: error: Unable to find a destination matching the provided destination specifier:
    		{ id:3F7D1564-79BA-40B5-8CFC-2D0E81F2D099 }
    
    	The requested device could not be found because no available devices matched the request.
    
    	Available destinations for the "App" scheme:
    		{ platform:watchOS, id:dvtdevice-DVTiOSDevicePlaceholder-watchos:placeholder, name:Any watchOS Device }
    		{ platform:watchOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-watchsimulator:placeholder, name:Any watchOS Simulator Device }
    		{ platform:watchOS Simulator, id:3F7D1564-79BA-40B5-8CFC-2D0E81F2D099, OS:9.0, name:Apple Watch SE (40mm) (2nd generation) }
                ...
```

Ummmm it's right there 🤔

**Pull request merged**